library/RevAPI/Rev.php

```diff
@@ -25,20 +32,23 @@ class Rev {
      * @param string $user_api_key
      * @param string|null $host - if null, it will use the default production host
      * @param array $http_config - array of guzzle http configuration options
+     * @param array $sandbox_mode - boolean of whether to use sandbox mode
      */
-    public function __construct($client_api_key, $user_api_key, $host = null, $http_config = array())
+    public function __construct($client_api_key, $user_api_key, $host = null, $http_config = array(), $sandbox_mode = false)
     {
-        if (null == $host) {
-            $host = self::PRODUCTION_HOST;
+        // set sandbox mode by host or sandbox mode (hosts supports backwards compatibility)
+        if ($host === self::SANDBOX_HOST) {
```

Comment:
Not sure I understand the point of this when SANDBOX_HOST has been changed to www.rev.com/api/sandbox. (In practice the sandbox backwards compat becomes moot in April so it doesn't really matter I guess.)

Reply:
The backward compatibility comes from the use and existence of the `\RevAPI\Rev::SANDBOX_HOST` constant. If you use it will behave correctly.  We can make a breaking change to the RevAPI constructor
from `public function __construct($client_api_key, $user_api_key, $host = null, $http_config = array())` to `public function __construct($client_api_key, $user_api_key, $http_config = array(), $sandbox_mode = false)` instead of current change to `public function __construct($client_api_key, $user_api_key, $host = null, $http_config = array(), $sandbox_mode = false)`.